src/pages/components/parallax.mdx

```diff
@@ -3,7 +3,55 @@
 ## Overview
 
 ```js
-import { Parallax } from 'react-spring'
+import { Parallax, ParallaxLayer } from 'react-spring'
 ```
 
-> 🚧 Oh no, that's embarrasing. It looks like we don't have the knowledge to fill this page in. If you think you do, please consider submitting a [PR](https://github.com/pmndrs/react-spring.io) 🚧
+`Parallax` creates a scrollable container. `ParallaxLayer`s contain your content and will be moved according to their offsets and speeds.
+
+**NOTE**: Currently, only `@react-spring/web` is supported.
+
+```js
+<Parallax ref={ref} pages={3} horizontal>
+  <ParallaxLayer offset={0} speed={0.5}>
+    <span
+      onClick={() => {ref.current.scrollTo(1)}}
+    >
+      Layers can contain anything
+    </span>
+  </ParallaxLayer>
+</Parallax>
+```
+
+## Parallax
+
+**Props**:
```

Comment:
maybe these and the ones below should be a table like https://react-spring.io/common/props ?

Reply:
Agree, much prefer tables. I went with list first because of the README.